thirdparty/hairer/radau_decsol_c.c

```diff
@@ -1107,6 +1110,9 @@ static doublereal c_b116 = .25;
     } else {
 		/* --- COMPUTE JACOBIAN MATRIX ANALYTICALLY */
     	(*jac)(n, x, &y[1], &fjac[fjac_offset], ldjac, &rpar[1], &ipar[1], jac_PY);
+		if (ipar[1] < 0){
+			goto L79;
+		}

```

Comment:
Double indentation

Reply:
This is a tab, but by default GitHub renders tabs as 8 spaces, which makes it look like 2 tabs. (Can change that under Profile->settings->Appearence).

Any action needed on this, or are we fine with a mixture of 4 spaces and tabs?